app/services/katello/pulp3/content_view_version/importable_repositories.rb

```diff
@@ -2,13 +2,18 @@ module Katello
   module Pulp3
     module ContentViewVersion
       class ImportableRepositories
-        attr_accessor :creatable, :updatable, :organization, :metadata
+        attr_accessor :creatable, :updatable, :organization, :metadata, :redhat
```

Comment:
I think these should be attr_reader ?

Reply:
Agreed. Make em attr_reader and just set `@organization = organization` etc in the constructor.